Content.Shared/CPR/SharedCPRSystem.cs

```diff
@@ -0,0 +1,115 @@
+using Content.Shared.Damage;
+using Content.Shared.Mobs;
+using Content.Shared.Mobs.Components;
+using Content.Shared.Verbs;
+using Robust.Shared.Utility;
+using Content.Shared.DoAfter;
+using Content.Shared.Interaction;
+using Robust.Shared.Serialization;
+using Robust.Shared.Audio.Systems;
+using Robust.Shared.Network;
+using Robust.Shared.Timing;
+using Content.Shared.Popups;
+using Content.Shared.IdentityManagement;
+
+namespace Content.Shared.CPR;
+/// <summary>
+/// Used for handling CPR on critical breathing mobs
+/// </summary>
+public abstract partial class SharedCPRSystem : EntitySystem
+{
+    [Dependency] private readonly DamageableSystem _damage = default!;
+    [Dependency] private readonly SharedInteractionSystem _interactionSystem = default!;
+    [Dependency] private readonly SharedDoAfterSystem _doAfter = default!;
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+    [Dependency] protected readonly EntityManager Ent = default!;
+    [Dependency] private readonly INetManager _net = default!;
+    [Dependency] protected readonly IGameTiming Timing = default!;
+    [Dependency] private readonly SharedPopupSystem _popup = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<CPRComponent, GetVerbsEvent<ExamineVerb>>(OnGetExamineVerbs);
+        SubscribeLocalEvent<CPRComponent, CPRDoAfterEvent>(OnDoCPREvent);
+    }
+    public void OnDoCPREvent(Entity<CPRComponent> ent, ref CPRDoAfterEvent args)
```

Comment:
```suggestion
    }

    public void OnCPRDoAfter(Entity<CPRComponent> ent, ref CPRDoAfterEvent args)
```
fix spacing and naming conventions